resources/public/api/conf/P1.0/schemas/contact-details-response.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Contact Details",
+  "type": "object",
+  "properties": {
+    "_links": {
+      "type": "object",
+      "description": "A list of endpoint links that indicate possible actions related to the current resource.",
+      "properties": {
+        "self": {
+          "type": "object",
+          "description": "The Uniform Resource Identifier (URI) for the endpoint to get addresses for the requested match ID. ",
+          "properties": {
+            "href": {
+              "type": "string",
+              "example": "/individuals/details/contact-details?matchId=57072660-1df9-4aeb-b4ea-cd2d7f96e430"
+            }
+          }
+        }
+      }
+    },
+    "contactDetails" : {
+      "type": "array",
+      "description": "List of contact details for the individual",
+      "properties": {
+        "detail": {
+          "type": "string",
+          "description": "String of information related to type",
+          "example": "fred.blogs@hotmail.com"
```

Comment:
Again concerned about using emails when email has not been requested by the OGDs